src/index.html

```diff
@@ -9,7 +9,66 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
+<form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST">
+  <fieldset class="mb20">
+    <legend>Personal information</legend>
+    <label class="mb10">
+      Surname:<input type="text" name="surname" autocomplete="off"></label>
+    <label class="mb10">
+      Name:<input type="text" name="name" autocomplete="off"></label>
+    <label class="mb10">
+      How old are You?<input type="number" name="age" min="12" max="100" value="12"></label>
+    <label class="mb10">
+      Full date of birth:<input type="date" name="date-of-birth"></label>
+    <label class="mb10 mb0">
+      I accept the term of the agreement<input type="checkbox" name="agreement" required></label>
+  </fieldset>
+
+  <fieldset class="mb20">
+    <legend>Registration</legend>
+    <label class="mb10">
+      E-mail:<input type="email" name="email" placeholder="email@example.com" required></label>
+    <label class="mb10 mb0">
+      Password:<input type="password" name="password" minlength="8" maxlength="16" required></label>
+  </fieldset>
+
+  <fieldset class="mb20">
+    <legend>An interesting fact about you!</legend>
+    <div class="mb10">
+      Do you love cats?
+      <label><input type="radio" name="love-cats" value="cat-yes">Yes</label>
```

Comment:
One tag - one line. Use new lines for nested tags. Don't write nested tags in one line.